### ISSUE TITLE
R CMD check with dev version of ggplot2

### ISSUE THREAD

**Issue by hadley** [dgrtwo/broom#90]
I see:

```
checking examples ... ERROR
Running examples in ‘broom-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: cch_tidiers
> ### Title: tidiers for case-cohort data
> ### Aliases: cch_tidiers glance.cch tidy.cch
> 
> ### ** Examples
> 
> if (require("survival", quietly = TRUE)) {
+     # examples come from cch documentation
+     subcoh <- nwtco$in.subcohort
+     selccoh <- with(nwtco, rel==1|subcoh==1)
+     ccoh.data <- nwtco[selccoh,]
+     ccoh.data$subcohort <- subcoh[selccoh]
+     ## central-lab histology
+     ccoh.data$histol <- factor(ccoh.data$histol,labels=c("FH","UH"))
+     ## tumour stage
+     ccoh.data$stage <- factor(ccoh.data$stage,labels=c("I","II","III" ,"IV"))
+     ccoh.data$age <- ccoh.data$age/12 # Age in years
+ 
+     fit.ccP <- cch(Surv(edrel, rel) ~ stage + histol + age, data = ccoh.data,
+                    subcoh = ~subcohort, id= ~seqno, cohort.size = 4028)
+ 
+     tidy(fit.ccP)
+ 
+     # coefficient plot
+     library(ggplot2)
+     ggplot(tidy(fit.ccP), aes(x = estimate, y = term)) + geom_point() +
+         geom_errorbarh(aes(xmin = conf.low, xmax = conf.high), height = 0) +
+         geom_vline()
+ 
+     # compare between methods
+     library(dplyr)
+     fits <- data_frame(method = c("Prentice", "SelfPrentice", "LinYing")) %>%
+         group_by(method) %>%
+         do(tidy(cch(Surv(edrel, rel) ~ stage + histol + age, data = ccoh.data,
+                     subcoh = ~subcohort, id= ~seqno, cohort.size = 4028,
+                     method = .$method)))
+ 
+     # coefficient plots comparing methods
+     ggplot(fits, aes(x = estimate, y = term, color = method)) + geom_point() +
+         geom_errorbarh(aes(xmin = conf.low, xmax = conf.high)) +
+         geom_vline()
+ }

Attaching package: ‘dplyr’

The following objects are masked from ‘package:stats’:

    filter, lag

The following objects are masked from ‘package:base’:

    intersect, setdiff, setequal, union

Error: geom_vline requires the following missing aesthetics: xintercept
Execution halted
```

Can you please fix ASAP? I'm submitting to CRAN on Friday

**Comment by dgrtwo** [comment]
This is fixed in broom 0.4.0 (f67a8b989334ee2a1d4d943a40700462778a5144), submitted to CRAN today.

**Comment by dgrtwo** [comment]
broom 0.4.0 on CRAN now.
